### PR TITLE
Migrate FileUpload sample to ASP.NET Core 2x

### PR DESCRIPTION
**Pull request by teamkiller7112**
Migrate FileUpload sample to ASP.NET Core 2.0.

Disable request size limit for StreamingController by adding DisableRequestSizeLimit attribute.

### CODE REVIEW

**Comment by user135711** [comment]
General feedback for the sample, not the pull request: I think the sample should actually save the uploaded image to a public folder or even better a database and then display it in the view in ProfileController.cs Line63 mainly because I want to learn how to do something similar with pdfs and books. I checked out this branch and it ran fine on my machine. Here's how I fixed the errors 2 weeks ago for [comparison](https://github.com/user135711/Docs/tree/FileUploadMigration/aspnetcore/mvc/models/file-uploads/sample/FileUploadSample).

File: aspnetcore/mvc/models/file-uploads/sample/FileUploadSample/FileUploadSample.csproj [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <OutputType>Exe</OutputType>
+    <PackageId>FileUploadSample</PackageId>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="wwwroot\**\*;Views\**\*;Areas\**\Views">
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.2.1" />
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/models/file-uploads/sample/FileUploadSample/FileUploadSample.csproj]: Use Microsoft.AspNetCore.App  see https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-2.1

**Comment by guardrex** [comment]
btw - This is due for a major overhaul on https://github.com/aspnet/Docs/issues/5331.

**Comment by guardrex** [comment]
😅 Finally have a chance to take a closer look here. We're going not going to be able to accept this because this makes a set of updates to the 1.0 sample that seek to convert it directly to 2.1 and we must maintain a 1.1 version of the sample with the topic. A PR that seeks to provide a 2.1 sample for the topic will require two versions of the sample and associated topic updates to render their code snippets. There are also a number of changes that are required beyond the updates here in order to make this conform to our 2.1 template.

This PR is linked now on the tracking issue (#5331) for the topic+sample, so these changes will be considered when that issue is worked. Thanks @teamkiller7112 for providing this.

**Pull request closed without merging**